skglm/tests/test_estimators.py

```diff
@@ -228,6 +227,15 @@ def test_grid_search(estimator_name):
                                ours_clf.best_params_["alpha"], rtol=1e-3)
 
 
+@pytest.mark.parametrize("estimator_name", ["Lasso", "SVC"])
```

Comment:
is there a reason not to test the warm start for all the estimators?

Reply:
I just wanted the test to be fast. I was thinking if it worked for one estimator it would work for all of them

Reply:
OK, I just added all the estimators to the test!